WordPress/Classes/Models/Blog.h

```diff
@@ -175,8 +175,8 @@ typedef NS_ENUM(NSInteger, SiteVisibility) {
 @property (nonatomic, strong, readwrite, nullable) NSNumber *quickStartTypeValue;
 
 // Site Switcher Convenience Properties
-@property (nonatomic, strong, readwrite, nullable) NSDate *pinnedDate;
-@property (nonatomic, strong, readwrite, nullable) NSDate *lastUsed;
+@property (nonatomic, strong, readwrite, nullable) NSDate *pinnedDate DEPRECATED_ATTRIBUTE;
+@property (nonatomic, strong, readwrite, nullable) NSDate *lastUsed DEPRECATED_ATTRIBUTE;
```

Comment:
What do you think moving deprecated properties into the .m file? They'll continue to exist in the database, but no one can use them.

Reply:
It should be safe to simply remove these, which I've done now.
There are a few other unused fields in the DB. We'll need to clean these up at some point if there is a new migration needed.